contributor/06-conclusion-article.asciidoc

```diff
@@ -1,18 +1,18 @@
 == Conclusion
 
-Thanks for watching the Contributor segment of the InnerSource Commons Learning Path. With this section you've learned about the Contributor role - the life blood of InnerSource projects: External to the owners of a component's team, they bring additional valuable input to the project.
+Thanks for reviewing the Contributor segment of the InnerSource Commons Learning Path. With this section you've learned about the Contributor role - the life blood of InnerSource projects: external to the component owners' team, they bring additional valuable input to the project.
```

Comment:
```suggestion
Thanks for reviewing the Contributor segment of the InnerSource Commons Learning Path. With this section you've learned about the Contributor role - the life blood of InnerSource projects.  Contributors are external to the component owners' team, and they bring additional valuable input to the project.
```

Reply:
I'm not sure how to separate the PRs, since I did the mechanics article first, followed by conclusion. Can we please review the mechanics first, agree on that (or not), and then I can sync any changes from the prior PR into this one?
Or please let me know what's the best way to submit "just" this article change, and then not override the other PR?

Reply:
Likely you'd have to do an [interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) to remove the commit affecting the conclusion.
I think trying to merge the other pull request into this one before merging it might be a route that _could_ work using the web UI.

Reply:
Merging #198 first is my hope (or else I need to spend some time learning what is interactive rebase and how not to screw it up; which I can do but not until the weekend).
I committed @rrrutledge 's suggestions in #198 and answered about the link - please opine on the link (so we agree or I will change it back), and if no other comments on #198, would someone please approve it?

Reply:
Approved #198